_posts/2020-08-26-engineering-firstweek.md

```diff
@@ -0,0 +1,23 @@
+
+Thankfully I'm not alone at first day orientation, I'm joined by a group of new hires. Since we are all in the same boat we've started our own new-hire slack channel to support each other. To get started there is a helpful and thorough checklist for us newbies to walkthrough for setting up the basics. The better part is the team specific onboarding guide that is focused on setup tasks that pertain to tools I'll use for everyday type work. The best part though, has got to be that I got a new buddy! Scribd does new-hire buddies to show you the ropes. It goes a long way to take a little bit of the new-job tension off. You can go to your new hire buddy for anything from onboarding to team specific questions. In my case my buddy is a tech lead so he has a wealth of knowledge on our stack or knows who to point you to if it isn't in their domain.
+
+I was happy to hear that Scribd does new hire meet and greets to introduce managers and key people from other teams to start building collaborative relationships. Typically they are from teams you'd work closely with so that you can add more faces to your cache and get to know people. This is huge in getting comfortable with the team's area of responsibilities and who knows what. Everyone I've met at Scribd is crazy smart but more importantly super friendly which makes growing into the new job much smoother and easier. There are fun Slack channels for all kinds of things like coffee, cooking, pets, sports, cars, you name it making it easy to nerd out and meet new people.
+
+As for the tech stack, some of it is completely new to me but that makes it exciting to learn new things and get different perspectives on solving problems. It also makes for many learning opportunities from the folks who are SMEs on areas of the stack. Being on the core infrastructure team I get to keep up with Terraform, AWS, Kubernetes and fresh new tech that comes. Our team builds and maintains the infrastructure foundation that is shared by the other engineering teams. For me the cool thing is being part of many engineering efforts and getting to share and learn knowledge from each team and each project.
+
+My first day had other exciting news too! Scribd announced the acquisition of Slideshare, which, in current times it is great to see Scribd is doing well and making progress even as so much of the world faces trying challenges.
```

Comment:
```suggestion
Any first week at Scribd is going to be memorable but I was especially lucky in that my first day [Scribd announced the acquisition of SlideShare](https://blog.scribd.com/home/welcome-slideshare-to-the-scribd-community). I can't guarantee that your first day will be _that_ exciting, but I'm certain [your first week will be a good one](https://tech.scribd.com/careers/#open-positions).
```

The end of the article was a little abrupt, so I tried to ease the reader out a little more.

The "challenges" line was a little bit of a bummer, so I cut it to try to end a little more upbeat :smile: